To: vim_dev@googlegroups.com
Subject: Patch 7.3.394
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.394
Problem:    When placing a mark while starting up a screen redraw messes up
	    the screen. (lith)
Solution:   Don't redraw while still starting up. (Christian Brabandt)
Files:	    src/screen.c


*** ../vim-7.3.393/src/screen.c	2011-09-02 14:07:31.000000000 +0200
--- src/screen.c	2012-01-10 12:36:52.000000000 +0100
***************
*** 764,772 ****
  	    doit = TRUE;
      }
  
!     /* Return when there is nothing to do or screen updating already
!      * happening. */
!     if (!doit || updating_screen)
  	return;
  
      /* update all windows that need updating */
--- 764,776 ----
  	    doit = TRUE;
      }
  
!     /* Return when there is nothing to do, screen updating is already
!      * happening (recursive call) or still starting up. */
!     if (!doit || updating_screen
! #ifdef FEAT_GUI
! 	    || gui.starting
! #endif
! 	    || starting)
  	return;
  
      /* update all windows that need updating */
*** ../vim-7.3.393/src/version.c	2012-01-04 20:29:18.000000000 +0100
--- src/version.c	2012-01-10 12:41:32.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     394,
  /**/

-- 
It is illegal for anyone to try and stop a child from playfully jumping over
puddles of water.
		[real standing law in California, United States of America]

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
